main/src/cgeo/geocaching/AboutActivity.java

```diff
@@ -239,6 +243,20 @@ public void setContent() {
         }
     }
 
+    private static String prepareChangelogBugfix(final Activity activity) {
+        String changelog = FileUtils.getChangelogRelease(activity).trim();
+        int current = 0;
+        final int max = BUGFIX_VERSION_NAME.length;
+        final Pattern p = Pattern.compile("((^|[\\r\\n]+)##[\\r\\n]+)");
+        Matcher match = p.matcher(changelog);
+        while (match.find() && current < max) {
+            changelog = changelog.substring(0, match.start(0)) + "\r\n## " + BUGFIX_VERSION_NAME[current] + " Bugfix Release\r\n" + changelog.substring(match.end(0));
```

Comment:
Just a reminder, that the string `Bugfix Release` should be externalized for translation. But I think you are aware of that, just still WIP.

Reply:
done